src/index.html

```diff
@@ -1,19 +1,456 @@
+                <div id="learn-more3" class="service__item--learn-more-button">
+                  <img src="./images/services/learn-more.png" alt="learn-more">
+                </div>
+                <label for="learn-more3">Learn more</label>
+              </div>
+            </div>
+            <div class="service__item-4">
+              <div class="service__item--number">
+                <p>004</p>
+              </div>
+              <div class="service__item--jobs">
+                Digital Communications<br>
+                Influencer Marketing<br>
+                Product Placements<br>
+                Strategic Partnerships
+              </div>
+              <div class="service__item--learn-more">
+                <div id="learn-more4" class="service__item--learn-more-button">
+                  <img src="./images/services/learn-more.png" alt="learn-more">
+                </div>
+                <label for="learn-more3">Learn more</label>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+      <!-- #endregion -->
+      <!-- #region Testimonials -->
+      <div class="container">
+        <section class="testimonials block-margin" id="testimonials">
+          <p>
+            Testimonials
+          </p>
+          <div class="testimonials__title">
+            What people&nbsp;say
+          </div>
+          <div class="testimonials__cards">
+
+            <article class="testimonials__card">
+              <div class="testimonials__card-photo">
+                <img src="./images/testimonials/testimonial-ava-1.jpg" alt="avatar">
+              </div>
+              <div class="testimonials__card-comment">
+                <img src="./images/testimonials/“.png" alt="image">
+                AIR’s ideas are refreshing and out of the box. Authentic team that focuses on the important path of the brand.
+              </div>
+              <div class="testimonials__card-name-professional--wrapper">
+                <div class="testimonials__card-name">
+                  Tal Gilad
+                </div>
+                <div class="testimonials__card-profession">
+                  Teach for America
+                </div>
+              </div>
+            </article>
+            <article class="testimonials__card">
+              <div class="testimonials__card-photo">
+                <img src="./images/testimonials//testimonial-ava-2.jpg" alt="avatar">
+              </div>
+              <div class="testimonials__card-comment">
+                <img src="./images/testimonials/“.png" alt="image">
+                AIR is an exceptional agency that leads with creative talent, first-class account servicing.
+              </div>
+              <div class="testimonials__card-name-professional--wrapper">
+                <div class="testimonials__card-name">
+                  Azadeh Hawkins
+                </div>
+                <div class="testimonials__card-profession">
```

Comment:
bad usage of html tags, this should be a paragraph or a span.

(i won't comment on the another ones, please fix all related issues on the html document)